docs/transfer-ledger-create.html

```diff
@@ -0,0 +1,1092 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <script>
+
+      //Form Validation
+      $(document).ready(function(){
+
+       // custom form validation rule
+       $.fn.form.settings.rules.validSig = function(sig) {
+         if (sig.length === 128 && !isPriv(sig)) {
+           return validateSig(sig);
+         }
+         else return true;
+       };
+
+       $.fn.form.settings.rules.validPriv = function(sig) {
+         if (sig.length === 128 && isPriv(sig)){
+           return validatePriv(sig.slice(0,64))
+         } else if (sig.length === 64) {
+           return validatePriv(sig);
+         }
+         else return true;
+       };
+
+        $.fn.form.settings.rules.validEntry = function(sig) {
+          if (window.ledger)
+            return true;
+          if (sig.length === 128 || sig.length === 64) {
+            return true;
+          } else if (sig.length === 0){
+            if (document.getElementById("hash-string").innerText === ""){
+              return true;
+            } else return false;
+          }
+          else return false;
+        };
+
+       $.fn.form.settings.rules.checkKeyLength= function(key) {
+         if (document.getElementById("receiver-select").hidden === false){
+           if (document.getElementById("basic").classList.contains( 'active' )){
+             return key.length===64;
+           }
+           else return true;
+         }
+         else {
+           return true;
+         }
+       };
+
+       $.fn.form.settings.rules.checkCustomKeyLength= function(guard) {
+       if (document.getElementById("custom").classList.contains('active')){
+          guard = JSON.parse(guard);
+          if (guard.keys.length === 0) return false;
+          let validKey = guard.keys.filter(key => {
+            return key.length===64;
+          })
+          return validKey.length === guard.keys.length;
+        } else return true;
+      }
+
+       $.fn.form.settings.rules.checkPredicate = function(key) {
+         if (document.getElementById("receiver-select").hidden === false){
+           if (document.getElementById("custom").classList.contains( 'active' )){
+              let guard = JSON.parse(document.getElementById("custom-guard").value);
+              return (guard.pred === "keys-all" || guard.pred === "keys-any" || guard.pred === "keys-2");
+           } else return true;
+         }
+         else {
+           return true;
+         }
+       };
+
+       $.fn.form.settings.rules.checkKeyset = function(keyset) {
+         if (document.getElementById("receiver-exist").hidden === false){
+           keyset = JSON.parse(keyset)
+           if (validateKeyset(keyset)) return true;
+           else if (!!keyset.keysetref) return true;
+           else return false;
+         } else return true;
+       }
+       
+       $.fn.form.settings.rules.ledgerErrorPK = function(sig) {
+          //TO DO figure out a good rule for that
```

Comment:
Does this need to be addressed or can it be taken out?

Reply:
removed/unused